config.yaml

```
@@ -117,6 +117,7 @@ collate:
     restart: true
     mem: 4GB
     walltime: 1:00:00
+    mpi: true
```

```suggestion
    mpi: false.
```

We only need this to be true when using `mppnccombine-fast`. If this is inconsistent with the QA checks we'll need to update the checks as well.

---

My bad, I think I added the `mpi: true` line because I assumed the QA test code `assert config["collate"]["mpi"]` meant assert that the value was true rather than the value existed. I'll change this one.

---

Actually no, it does assert that it is `true`...https://github.com/ACCESS-NRI/access-esm1.5-configs/pull/44/checks?check_run_id=28688143378
See https://github.com/ACCESS-NRI/model-config-tests/blob/8d2e2ac5f04c36538d0c641416bc0a3d4b099fd1/src/model_config_tests/qa/test_access_esm1p5_config.py#L188-L190

---

Updated the tests as part of https://github.com/ACCESS-NRI/model-config-tests/pull/47